src/audio/aria/aria.c

```diff
@@ -318,29 +317,28 @@ static int aria_copy(struct comp_dev *dev)
 	sink_c = buffer_acquire(sink);
 
 	comp_get_copy_limits(source_c, sink_c, &c);
-	source_bytes = c.frames * c.source_frame_bytes;
-	sink_bytes = c.frames * c.sink_frame_bytes;
+	copy_bytes = MIN(c.frames * c.source_frame_bytes, cd->base.ibs);
```

Comment:
you don't need to take sink frame bytes into account at all?

Reply:
Aria supports only 32bit containers. Sink frame bytes is always equal source frame bytes.